jmxtrans-core/src/main/java/com/googlecode/jmxtrans/jobs/ServerJob.java

```diff
@@ -3,6 +3,7 @@
 import com.googlecode.jmxtrans.connections.JMXConnectionParams;
 import com.googlecode.jmxtrans.jmx.JmxUtils;
 import com.googlecode.jmxtrans.model.Server;
+import com.sun.tools.attach.VirtualMachine;
```

Comment:
Seems we are adding a dependency to the Sun / Oracle JVM. Not good. We need to at least encapsulate this so that it runs correctly on non Sun JVM.